main-character-syndrome/app.py

```
@@ -0,0 +1,236 @@
+
+
+@app.route("/api/demo", methods=["POST"])
+def api_demo():
+    payload = request.get_json(silent=True) or {}
+    scenario = payload.get("scenario", "mass_entry")
+    response = build_response({"scenario": scenario, "universe": payload.get("universe", "Indian Cinema")}, demo_scenario=scenario)
+    return jsonify(response)
+
+
+if __name__ == "__main__":
+    app.run(host="0.0.0.0", port=5000, debug=True)
```

Ruff (0.16.3)

[error] Possible binding to all interfaces

(S104)